src/CarbonAware.WebApi/src/Controllers/CarbonAwareController.cs

```diff
@@ -67,19 +67,23 @@ public async Task<IActionResult> GetBestEmissionsDataForLocationsByTime([FromQue
     [ProducesResponseType(StatusCodes.Status204NoContent)]
     [ProducesResponseType(StatusCodes.Status400BadRequest, Type = typeof(ValidationProblemDetails))]
     [HttpGet("bylocations")]
-    public async Task<IActionResult> GetEmissionsDataForLocationsByTime([FromQuery(Name = "location"), BindRequired] string[] locations, DateTime? time = null, DateTime? toTime = null, int durationMinutes = 0)
+    public async Task<IActionResult> GetEmissionsDataForLocationsByTime([FromQuery(Name = "location"), BindRequired] string[] locations, DateTimeOffset? time = null, DateTimeOffset? toTime = null, int durationMinutes = 0)
```

Comment:
Notes from meeting with @pritipath:

- Use constants and be explicit about `FromQuery(Name = LOCATION_CONSTANT)` when declaring controller method attributes